client/me/profile-links-add-other/index.jsx

```diff
@@ -126,40 +127,42 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<form className="profile-links-add-other" onSubmit={ this.onSubmit }>
-				<p>
-					{ this.translate( 'Please enter the URL and description of the site you want to add to your profile.' ) }
-				</p>
-				{ this.possiblyRenderError() }
-				<FormFieldset>
-					<FormTextInput
-						className="profile-links-add-other__value"
-						valueLink={ this.linkState( 'value' ) }
-						placeholder={ this.translate( 'Enter a URL' ) }
-						onFocus={ this.recordFocusEvent( 'Add Other Site URL Field' ) }
-					/>
-					<FormTextInput
-						className="profile-links-add-other__title"
-						valueLink={ this.linkState( 'title' ) }
-						placeholder={ this.translate( 'Enter a description' ) }
-						onFocus={ this.recordFocusEvent( 'Add Other Site Description Field' ) }
-					/>
-					<FormButton
-						className="profile-links-add-other__add"
-						disabled={ this.getFormDisabled() }
-						onClick={ this.recordClickEvent( 'Save Other Site Button' ) }
-					>
-						{ this.translate( 'Add Site' ) }
-					</FormButton>
-					<FormButton
-						className="profile-links-add-other__cancel"
-						isPrimary={ false }
-						onClick={ this.recordClickEvent( 'Cancel Other Site Button', this.onCancel ) }
-					>
-						{ this.translate( 'Cancel' ) }
-					</FormButton>
-				</FormFieldset>
-			</form>
+			<Card>
```

Comment:
I'm not sure about the Card within a Card look we generate with this. Do we have examples of that elsewhere?